sink-connector/src/test/kotlin/org/neo4j/connectors/kafka/sink/Neo4jCdcSchemaFromStreamsMessageIT.kt

```diff
@@ -158,6 +159,209 @@ class Neo4jCdcSchemaFromStreamsMessageIT {
     }
   }
 
+  @Neo4jSink(cdcSchema = [CdcSchemaStrategy(TOPIC)])
+  @Test
+  fun `should create a node with a null unique constraint property value`(
```

Comment:
would it make sense to add a test when the node is updated as well?

Reply:
Yep can add it.

Reply:
Added in latest commit.